app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+import random
+import matplotlib.pyplot as plt
+
+
+def flip_coin():
+    times = 10
+    heads_odds = {key: 0 for key in range(times + 1)}
+    for i in range(10000):
+        heads_count = 0
+        for j in range(10):
+            variant = random.randint(1, 2)
+            if variant == 1:
+                heads_count += 1
+        heads_odds[heads_count] += 1
+    for keys in heads_odds:
+        heads_odds[keys] = int((heads_odds[keys] / 10000) * 100)
+    return heads_odds
+
+
+def draw_gaussian_distribution_graph():
+    odds_dict = flip_coin()
+    drop = []
+    heads = []
+    for key, value in odds_dict.items():
+        drop.append(value)
+        heads.append(key)
+    plt.plot(heads, drop)
```

Comment:
You don't need 5 lines above the `plt.plot` because `dict` has methods `.keys()` and `.values()`.